### DISCUSSION TITLE
LuckPerms permission for going through blocks

### DISCUSSION THREAD

**Discussion by likeusb** [MinecraftFreecam/Freecam#215] [General]
Any idea if there's some sort of a specific permission I'd need to give to my players in order for them to be able to go through blocks in freecam without giving them op? Searched the list but nothing stands out in particular.

**Comment by MattSturgeon** [comment]
Currently Freecam doesn't support any granular server permissions, although we're open to doing so. If you have any particular systems in mine let us know or open an issue.

The way it works currently is that the build released on Modrinth restricts actions likely to be considered "cheating" to singleplayer and `op`, while the builds released elsewhere (github and curseforge) are unrestricted.

**Reply by likeusb** [reply]
I see, so downloading one of those versions would theoretically let the people just noclip through without op perms?

**Reply by MattSturgeon** [reply]
Yes.

As I said before, if there is a commonly used server permissions system then we'd be willing to add support on our end. I think https://github.com/MinecraftFreecam/Freecam/issues/105 is tracking that feature request.
